assets/css/index.css

```diff
@@ -42,151 +42,189 @@ main{
+  width: 35vw;
+  margin: auto;
 }
 .left-section h1{
-    font-size: xxx-large; 
-    font-weight: bolder;
+  font-size: xxx-large;
+  font-weight: bolder;
 }
 .subtitle{
-    padding-block: 10px;
-    font-weight: 400;
-    font-size:large;
+  padding-block: 10px;
+  font-weight: 400;
+  font-size: large;
 }
 .right-section img{
-    width: 100%;
-    height: 100%;
-    background-position: center;
-    background-size: contain;
+  width: 100%;
+  height: 100%;
+  background-position: center;
+  background-size: contain;
 }
 .heroBtn{
-    padding: 10px;
-    margin: 20px 0 10px 0;
-    color: white;
-    text-decoration: none;
-    border-radius: 5px;
-    display: inline-block;
-    transition: background .3s ease;
-    margin-right: 5px;
+  padding: 10px;
+  margin: 20px 0 10px 0;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  display: inline-block;
+  transition: background 0.3s ease;
+  margin-right: 5px;
 }
 .test{
-    background-color: #F9A826;
+  background-color: #f9a826;
 }
 .practice{
-    background-color: #0198E1;
+  background-color: #0198e1;
 }
 .test:hover{
-    background-color: #f9a826d5;
+  background-color: #f9a826d5;
 }
 .practice:hover{
-    background-color: #0196e1c7;
+  background-color: #0196e1c7;
 }
 
 #features{
-    justify-content: space-around;
-    text-align: center;
-    padding-block: 20px;
+  justify-content: space-around;
+  text-align: center;
+  padding-block: 20px;
 }
 .feature{
-    width: 25vw;
+  width: 25vw;
 }
 .feature img{
-    height: 96px;
-    width: 96px;
+  height: 96px;
+  width: 96px;
 }
 .feature p{
-    padding: 5px;
+  padding: 5px;
 }
 
-footer {
-    padding: 20px 0px;
-    font-weight: 500;
-    bottom: 0;
-    width: 100%;
-    font-size: 18px;
-    color: white;
-    display: flex;
-    justify-content: center;
-    background: rgb(2,0,36);
-background: linear-gradient(279deg, rgba(2,0,36,1) 0%, rgba(5,5,85,1) 52%, rgba(0,212,255,1) 100%);
-  }
+footer{
+  padding: 20px 0px;
+  font-weight: 500;
+  bottom: 0;
+  width: 100%;
+  font-size: 18px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  background: rgb(2, 0, 36);
+  background: linear-gradient(279deg, rgba(2, 0, 36, 1) 0%, rgba(5, 5, 85, 1) 52%, rgba(0, 212, 255, 1) 100%);
+}
 
 /* ============ Media Queries ============ */
 
-@media screen and (max-width:670px){
-    .nav-item{
-        display: inline-flex;
-    }
-    .navigation{
-        margin: auto;
-    }
-    .brand-name{
-        margin-bottom: 10px;
-    }
-    .brand{
-        margin: auto;
-    }
-    footer{
-        font-size: 12px;
-    }
+@media screen and (max-width: 525px) {
+  .navigation{
+    display: none;
+    position: absolute;
+    background-color: #0072ff;
+    top: 62px;
+    left: 0;
+    width: 100%;
+  }
+  ul.main-menu{
+    flex-direction: column;
+  }
+  .brand{
+    flex-grow: 1;
+  }
+  .mobile-nav{
+    display: block;
+    font-size: 25px;
+  }
+}
+
+@media screen and (max-width: 670px) {
```

Comment:
Add media query for `.mobile-nav` to hide from this checkpoint

because it causes error like this 
![image](https://user-images.githubusercontent.com/41796013/201521497-54371b72-cdcd-4347-a30a-3b814ed6d2c2.png)

Reply:
and also make the `.navigation` visible

Reply:
The nav-bar is fixed as requested